src/HtmlGenerator/Program.cs

```diff
@@ -26,18 +29,38 @@ static void Main(string[] args)
                     continue;
                 }
 
-                try
+                if (arg.StartsWith("/in:"))
                 {
-                    var project = Path.GetFullPath(arg);
-                    if (File.Exists(project))
+                    string inputPath = arg.Substring("/in:".Length).StripQuotes();
+                    try
                     {
-                        if (project.EndsWith(".sln", StringComparison.OrdinalIgnoreCase) ||
-                            project.EndsWith(".csproj", StringComparison.OrdinalIgnoreCase) ||
-                            project.EndsWith(".vbproj", StringComparison.OrdinalIgnoreCase))
+                        if (!File.Exists(inputPath))
                         {
-                            projects.Add(project.StripQuotes());
+                            continue;
+                        }
+                        using (StreamReader reader = new StreamReader(inputPath))

```

Comment:
I've made the change, although would you be able top explain why this would be preferred?